src/main/java/com/enderio/base/common/block/BuddingInfinityCrystalBlock.java

```diff
@@ -11,7 +11,6 @@
 //import net.minecraft.world.level.block.state.BlockState;
```

Comment:
You can just remove the infinity crystal stuff if you like, I don't believe it'll make it into the base mod